demo/index.html

```diff
@@ -8,7 +8,7 @@
     <body>
         <h1>Event Drops Demo</h1>
         <div id="eventdrops-demo" style="width: 90%;"></div>
-        <script src="./dist/d3.min.js"></script>
+        <script src="//d3js.org/d3.v4.js"></script>
```

Comment:
I think demo.js is bundling d3 . 
what I am not sure is if the dist/eventDrops.js is also bundling d3